For input string: "null" error in trying to build with gradle plugin.

**Summary**
I get this error once I update to 2.2.x from 1.0.2
For input string: "null" 

No further explanation seems to be available. 
No indication of whether this is from schema.json, fragments.graphql, queries.graphql. 

**Version**
2.2.2

**Description**
The exact cause is not known. I get this message during build. 

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:generateBciDebugServiceApolloSources'.
	....
Caused by: java.lang.NumberFormatException: For input string: "null"
	at com.apollographql.apollo.compiler.parser.GraphQLDocumentParser.normalizeValue(GraphQLDocumentParser.kt:724)
	at com.apollographql.apollo.compiler.parser.GraphQLDocumentParser.usedTypeDeclarations(GraphQLDocumentParser.kt:652)
	at com.apollographql.apollo.compiler.parser.GraphQLDocumentParser.parse(GraphQLDocumentParser.kt:27)
	at com.apollographql.apollo.gradle.internal.ApolloGenerateSourcesTask.taskAction(ApolloGenerateSourcesTask.kt:130)

*Add context about the problem here. How it happened and how to reproduce it:  
I am not sure.
It has a very large schema.json. 
There are 68 lines with "defaultValue": "null". I don't know which one it doesn't like. 
If I could find out how to find file and line where error occurs, I can post more information.

--- a/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/graphql/GraphQLDocumentParser.kt
+++ b/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/graphql/GraphQLDocumentParser.kt
@@ -732,9 +732,9 @@ class GraphQLDocumentParser(val schema: IntrospectionSchema, private val package
     return when (type.kind) {
       IntrospectionSchema.Kind.SCALAR -> {
         when (ScalarType.forName(type.name ?: "")) {
-          ScalarType.INT -> toString().trim().toInt()
-          ScalarType.BOOLEAN -> toString().trim().toBoolean()
-          ScalarType.FLOAT -> toString().trim().toDouble()
+          ScalarType.INT -> toString().trim().takeIf { it != "null" }?.toInt()
+          ScalarType.BOOLEAN -> toString().trim().takeIf { it != "null" }?.toBoolean()
+          ScalarType.FLOAT -> toString().trim().takeIf { it != "null" }?.toDouble()
           else -> toString()
         }
       }

